src/webui/api/rsscontroller.cpp

```diff
@@ -91,6 +94,29 @@ void RSSController::itemsAction()
     setResult(jsonVal.toObject());
 }
 
+void RSSController::markAsReadAction()
+{
+    requireParams({"itemPath"});
+
+    const QString itemPath {params()["itemPath"]};
+    const QString articleId {params()["articleId"]};
+
+    RSS::Item *item = RSS::Session::instance()->itemByPath(itemPath);
+    if (!item) return;
+
+    if (params().contains("articleId")) {
+        RSS::Feed *feed = qobject_cast<RSS::Feed *>(item);
+        if (feed) {
+            RSS::Article *article = feed->articleByGUID(articleId);
+            if (!article) return;
```

Comment:
It's meaningless to use `return` here since the following is just one line. Please write:
```c++
if (article)
    article->markAsRead();
```